modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js

```diff
@@ -987,36 +1005,25 @@ export default class GPUGridAggregator {
   }
 
   setupModels(fp64 = false) {
-    if (this.gridAggregationModel) {
-      this.gridAggregationModel.delete();
-    }
-    this.gridAggregationModel = this.getAggregationModel(fp64);
+    this.setupAggregationModel(fp64);
     if (!this.allAggregationModel) {
       // Model doesn't have to change when fp64 flag changes
       this.allAggregationModel = this.getAllAggregationModel();
     }
   }
 
+  setupAggregationModel(fp64 = false) {
```

Comment:
Why create a (public) method for a function that is only 3 lines? Same for `getAggregationModel` and `getAllAggregationModel` - these negatively impact the bundle size. Either merge them or make them utility functions outside of this class.

Reply:
Took pass on class methods, moved several into helper methods, will make more deeper look in future aggregator specific PRs.